pages-sections/about-us/TeamMemberCard.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import Card from "../../components/Card/Card";
+import CardAvatar from "../../components/Card/CardAvatar";
+import CardBody from "../../components/Card/CardBody";
+import Button from "../../components/CustomButtons/Button";
+import Popover from '@material-ui/core/Popover';
+import Typography from '@material-ui/core/Typography';
+
+const TeamMemberCard = ({ classes, member}) => {
+  const [anchorEl, setAnchorEl] = React.useState(null);
```

Comment:
By having this logic inside this component, each instance will have all of this logic bound to itself (rather than having the logic in "SectionTeam" and having to keep track of which popover we want to open)

We want a component to be able to "live on its own" when possible, and this setup allows that. We can use this component anywhere, and it will come with "popover" capability without any further code in the consumer.